packages/stokhos/test/UnitTest/Stokhos_HermiteBasisUnitTest.cpp

```diff
@@ -61,7 +61,7 @@ namespace HermiteBasisUnitTest {
     OrdinalType p;
     Stokhos::HermiteBasis<OrdinalType,ValueType> basis;
     
-    UnitTestSetup() : rtol(1e-12), atol(1e-7), p(10), basis(p) {}
+    UnitTestSetup() : rtol(1e-12), atol(1e-5), p(10), basis(p) {}
```

Comment:
Consider doing a bit of numerical analysis later (not for this PR) to get these tolerances.